clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -1114,18 +1113,18 @@ public MetadataResponse getMetadata(String storeName) {
         if (resource.endsWith("v" + store.getCurrentVersion())) {
           for (Partition partition: customizedViewRepository.getPartitionAssignments(resource).getAllPartitions()) {
             List<CharSequence> instances = new ArrayList<>();
-            for (ReplicaState replicaState: customizedViewRepository.getReplicaStates(resource, partition.getId())) {
-              if (replicaState.getVenicePushStatus().equals(ExecutionStatus.COMPLETED.name())) {
-                instances.add(replicaState.getParticipantId());
-              }
+            for (Instance instance: customizedViewRepository.getReadyToServeInstances(resource, partition.getId())) {
```

Comment:
Since there are changes in `KafkaStoreIngestionService`, the PR title should include `[server]` and `[dvc]` too

Reply:
Can you also add the `[server]` tag